src/pages/RegistrationPage.js

```diff
@@ -0,0 +1,32 @@
+import BasePage from '../base/BasePage'
+
+export default class RegistrationPage extends BasePage {
```

Comment:
I suspect you meant `RegistrationPopup`

it's not a page but a component

it should not extend BasePage